phpBB/phpbb/search/backend/fulltext_sphinx.php

```diff
@@ -11,7 +11,12 @@
 *
 */
 
-namespace phpbb\search;
+namespace phpbb\search\backend;
+
+use phpbb\auth\auth;
+use phpbb\config\config;
+use phpbb\event\dispatcher_interface;
+use phpbb\user;
 
 define('SPHINX_MAX_MATCHES', 20000);
```

Comment:
Are these defines needed for sphinx? Otherwise I'd opt for getting rid of these and use class constants where possible.